certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2934 @@
+      "description": "Type of data source"
+    },    
+    "BaselineData": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "startDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "dueDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "duration": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "effort": {
+          "format": "double",
+          "type": "number"
+        },
+        "originalEstimate": {
+          "format": "double",
+          "type": "number"
+        }
+      }
+    },
+    "TaskAttributes": {
+      "x-ms-summary": "Task fields",
+      "description": "Task fields (e.g. {\"Name\": \"My Name\", \"Description\": \"TBD\"})",
+      "type": "object",
+      "properties": {},
+      "additionalProperties": {}
+    },
+    "TaskResponse": {
+      "description": "Task details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "Task Id",
+          "description": "Unique Task Identifier",
+          "type": "string"
+        },
+        "attributes": {
+          "$ref": "#/definitions/TaskAttributes"
+        },
+        "sourceType": {
+          "$ref": "#/definitions/SourceType"
+        },
+        "baseline": {
+          "$ref": "#/definitions/BaselineData"
+        }
+      }
+    },    
+    "TaskResponseListTask": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "x-ms-summary": "Tasks",
+          "description": "List of tasks",
+          "items": {
+            "$ref": "#/definitions/TaskResponse"
+          }
+        }
+      }
+    },
+    "TaskResponseResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/TaskResponse"
+        }
+      }
+    },
+    "UpdateChallengeRequest": {
+      "description": "Challenge Update model",
+      "type": "object",
+      "properties": {
+        "updates": {
+          "$ref": "#/definitions/ChallengeAttributes"
+        }
+      }
+    },
+    "UpdateIdeaRequest": {
+      "description": "Idea Update model",
+      "type": "object",
+      "properties": {
+        "updates": {
+          "$ref": "#/definitions/IdeaAttributes"
+        }
+      }
+    },
+    "TokenInfo": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "Token Id",
+          "description": "Unique Token Identifier",
+          "type": "string"
+        },
+        "name": {
+          "type": "string",
+          "x-ms-summary": "Token Name",
+          "description": "Name of the API Token"
+        },
+        "expirationDate": {
+          "format": "date-time",
+          "x-ms-summary": "Token Exp Date",

```

Comment:
@vsorokin-ppmexpress please make sure everything is grammatically correct and is in complete form. You can refer to [English strings guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings). If the summary will be like `Token Expiration Date`, then we also need to change the description below so that they don't look like it's a copy-paste.